metron-deployment/amazon-ec2/README.md

```diff
@@ -41,11 +50,13 @@ Any platform that supports these tools is suitable, but the following instructio
   /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
   ```
 
-2. With Homebrew installed, run the following command in a terminal to install all of the required tools.
+2. With Homebrew installed, run the following command in a terminal to install all of the required tools and dependencies.
 
   ```
-  brew cask install java
-  brew install maven git
+  brew update
```

Comment:
I would be concerned with suggesting such a broad-reaching command.

Reply:
`brew update` is effectively the same as `apt-get update`. It doesn't upgrade all installed packages.

Reply:
Apologies, you're right - I misread it as `upgrade`.

Reply:
np, thx for reviewing.